### PR TITLE
Contributors Section and Socalify added

### PR DESCRIPTION
**Pull request by PeritusDeveloper**
## What is this Pull Request About?
- PR is related to the  #107.
- Added Contributors and made readme more appealing

## What will this Pull Request Affect?

- The new Contributing will have better view about the project
- The contributors would be acknowledged after there PR's are merged by mention them in README file
- The Contributors list is automatically updated
- Added Copyright Logo in README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,4 +61,23 @@ search("Hello, World!", Engine.Google)
...
+<a href="https://github.com/ArjunSharda/Searchor/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=ArjunSharda/Searchor&max=100&columns=11" width=97%/>
+</a>
+</div>
+<br>
+<br>
+<hr>
+<h6 align="center">© Searchor 2022 
+<br>
+All Rights Reserved</h6>
```
Comment (ArjunSharda) [inline README.md:83]:
```suggestion
## Want to contribute?
Take a look at the [contributing guidelines](CONTRIBUTING.md)!
<hr>
<h6 align="center">© Arjun Sharda 2022 
<br>
All Rights Reserved</h6>

**Review by ArjunSharda — CHANGES_REQUESTED** [review]
Awesome! Do you mind pushing these changes i've requested? I think the contributors page would eventually as the project gets more well known be very big and would flood the README. I've also edited the contributors section. Thank you so much for this change by the ways.

**Comment by PeritusDeveloper** [comment]
I have updated it accordingly
Please review!

File: README.md (hunk 1 above)
Comment (PeritusDeveloper) [reply README.md]: You have a valid point.
I have changed it

**Pull request merged**